.babelrc

```diff
@@ -10,6 +10,7 @@
     },
     "production": {
       "plugins": [
+        "transform-react-remove-prop-types",
```

Comment:
But this is for the minified/production build. This shouldn't be used for development anyway, should it? Or are you not planning on releasing a regular and a minified bundle?